images/linux/scripts/installers/aws-sam-cli.sh

```diff
@@ -2,15 +2,31 @@
 ################################################################################
 ##  File:  aws-sam-cli.sh
 ##  Desc:  Installs AWS SAM CLI
-##         Must be run as non-root user after homebrew and clang
+##         Requires Pyton >=3.6, must be run as non-root user after toolset installation
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install aws sam cli
-brew tap aws/tap
-brew install aws-sam-cli
+# Download latest aws sam cli sources
+TarballUrl=$(curl -s https://api.github.com/repos/awslabs/aws-sam-cli/releases/latest | jq -r '.tarball_url')
+TarballPath="/tmp/aws-sam-cli.tar.gz"
```

Comment:
I think our bash variables naming convention is either ALL_UPPER_CASE or all_lower_case symbols with underscores.

Reply:
Judging by the other scripts we don't have any convention 😀
I've seen both your options and Julia.sh contains the same name types
![image](https://user-images.githubusercontent.com/48208649/86339378-3034c800-bc5c-11ea-8f35-f7daf9161577.png)

Reply:
ok than, disregard my comment